examples/package-lock.json

```diff
@@ -1,84 +1,92 @@
 {
   "name": "examples-browser",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
I think this is the first time I've seen `lockfileVersion` set to 3. Does this mean we're bumping the minimum required node version with this update?

Reply:
Ah, I just saw the root `package.json` has the same version. So I guess it's fine.

Reply:
No. Lockfile 3 is backward compatible to version 7.

Reply:
Out of curiosity, what are you targeting these days ?  18 ?

CI looks like it's at 16 but other things look like 18.

Reply:
I don't think we really have an official policy on a min version for Node. 16 sounds reasonable I guess?

Reply:
Only a few weeks left of support on 16.  18 is on current Long Term Support.  20 is current.  

Highly recommend 18.

Diagram here. 
https://nodejs.dev/en/about/releases/

Reply:
OK, I'm fine with that.